### PR TITLE
i25-upgrade-solr

### PR DESCRIPTION
**Pull request by summer-cook**
ref ticket: https://github.com/scientist-softserv/oral-history/issues/25

Upgrades to solr 8.11
removes `<filter class="solr.StandardFilterFactory"/>` from the schema.xml file as the suggested replacement already exists `<filter class="solr.LowerCaseFilterFactory"/>` - documentation: https://stackoverflow.com/questions/56913835/upgrade-filter-in-schema-xml-with-solr-standardfilterfactory-from-existing-core

Video demo: https://share.getcloudapp.com/L1uDJ0j5

### CODE REVIEW

File: solr/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM solr:7.7
+FROM solr:8.1
```
Comment (orangewolf) [inline solr/Dockerfile:1]: this needs to be 8.11 I believe

**Review by orangewolf — CHANGES_REQUESTED** [review]
Not the right version to go to (8.11 is the only 8 version supported), also can you confirm this really works? we took solr down and brought the app up and imported some records with this setup?

**Comment by aprilrieger** [comment]
> 
Yes here is my workflow for testing:

get `.env.development` here: https://start.1password.com/open/i?a=LTLZ652TT5H5FHMYMASSH7PIXM&v=huuakin4bu4xanlhktv42qheam&i=ok6rwracafccnhrhur7usyfwoi&h=scientist.1password.com

gem install stack_car -v0.16.0
gem install dory

dory up
sc build
sc up

navigate to `http://oralhistory.test/users/sign_in`
login: creds in the .env

navigate to `http://oralhistory.test/admin`

run importer OR run in web container `rake db:migrate import[10]` [#] <— amount of works you import

Need troubleshooting tips?
If you do a `dc down -v`, odds are you will want to do a `rm -rf tmp/*`

**Comment by orangewolf** [comment]
if you open the solr admin interface does it show the solr 8 version? (just trying to be thorough)

**Comment by aprilrieger** [comment]
documentation `ERROR: Error CREATEing SolrCore 'blacklight-core': Unable to create core [blacklight-core] Caused by: solr.StandardFilterFactory`: https://github.com/projectblacklight/blacklight/pull/2017/files

**Comment by aprilrieger** [comment]
@orangewolf This is ready for re-review.

Note that I needed this PR merged into the branch for the linking to the correct field retriever on this [pr](https://github.com/scientist-softserv/oral-history/pull/31)
Demo video here: https://share.getcloudapp.com/L1uDJ0j5

**Review by orangewolf — APPROVED** [review]

**Comment by orangewolf** [comment]
@aprilrieger good work!

**Pull request merged**